docs/usage.md

```diff
@@ -176,16 +176,56 @@ maps", and provide one for each of the templates above.
 *   [Windows Autoscale Params][win-autoscale-params]
 *   [Windows Instance Params][win-instance-params]
 
-[lx-autoscale]: https://github.com/plus3it/watchmaker/blob/develop/docs/files/cfn/templates/watchmaker-lx-autoscale.template
-[lx-instance]: https://github.com/plus3it/watchmaker/blob/develop/docs/files/cfn/templates/watchmaker-lx-instance.template
-[win-autoscale]: https://github.com/plus3it/watchmaker/blob/develop/docs/files/cfn/templates/watchmaker-win-autoscale.template
-[win-instance]: https://github.com/plus3it/watchmaker/blob/develop/docs/files/cfn/templates/watchmaker-win-instance.template
+[lx-autoscale]: https://github.com/plus3it/watchmaker/blob/develop/docs/files/templates/lx-autoscale/watchmaker-lx-autoscale.template
+[lx-instance]: https://github.com/plus3it/watchmaker/blob/develop/docs/files/templates/lx-instance/watchmaker-lx-instance.template
+[win-autoscale]: https://github.com/plus3it/watchmaker/blob/develop/docs/files/templates/win-autoscale/watchmaker-win-autoscale.template
+[win-instance]: https://github.com/plus3it/watchmaker/blob/develop/docs/files/templates/win-instance/watchmaker-win-instance.template
 
 [lx-autoscale-params]: https://github.com/plus3it/watchmaker/blob/develop/docs/files/cfn/parameter-maps/watchmaker-lx-autoscale.params.json
 [lx-instance-params]: https://github.com/plus3it/watchmaker/blob/develop/docs/files/cfn/parameter-maps/watchmaker-lx-instance.params.json
 [win-autoscale-params]: https://github.com/plus3it/watchmaker/blob/develop/docs/files/cfn/parameter-maps/watchmaker-win-autoscale.params.json
 [win-instance-params]: https://github.com/plus3it/watchmaker/blob/develop/docs/files/cfn/parameter-maps/watchmaker-win-instance.params.json
 
+## `watchmaker` as a Terraform template
+
+Watchmaker can be integrated into a Terraform template as well. By wrapping
+the example CloudFormation templates within their respective Terraform template 
+they become deployable and manageable from within the [Terraform cli](https://www.terraform.io/). These templates 
+are intended as examples for you to modify and extend as you need.
+
+```eval_rst
+.. note::
+
+   * These templates assume that the accompanying CloudFormation template
+    is in the same directory.
+
+   * The links in this section are intended for viewing the templates
+    in a web browser. See the `Direct Downloads`_ section for links to the raw
+    files.
+```
+
+Variable values can be input interactively via the terraform console or 
+via a Terraform module. An example Terraform module that calls the 
+lx-autoscale template is shown below.
+
+```
+module "test-lx-instance" {
+  source = "./lx-autoscale"
```

Comment:
Let's set the source to `github.com/plus3it/watchmaker//docs/files/templates/lx-autoscale`. See https://www.terraform.io/docs/modules/sources.html#github. This way no one actually _has_ to download the files locally.